README.md

```
@@ -7,7 +7,7 @@
 * [Technologies Used](#technologies-used)
 * [Features](#features)
 * [Usage](#usage)
-* [Room for Improvement](#room-for-improvement)
+* [DEMO](#DEMO)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
Expected: #demo; Actual: #DEMO

(MD051, link-fragments)